app.py

```diff
@@ -22,7 +22,7 @@
 api.add_resource(DataSources, '/data-sources', resource_class_kwargs={'psycopg2_connection': psycopg2_connection})
 api.add_resource(DataSourceById, '/data-sources/<data_source_id>', resource_class_kwargs={'psycopg2_connection': psycopg2_connection})
 api.add_resource(Agencies, '/agencies/<page>', resource_class_kwargs={'psycopg2_connection': psycopg2_connection})
-api.add_resource(SearchTokens, '/search-tokens/<search>/<location>', resource_class_kwargs={'psycopg2_connection': psycopg2_connection})
+api.add_resource(SearchTokens, '/search-tokens/<endpoint>/<arg1>/<arg2>', resource_class_kwargs={'psycopg2_connection': psycopg2_connection})
```

Comment:
I think this only works for me when I do the quick-search route with two arguments. If I do the data-sources search with only one argument, it says that the route isn't found. I'm thinking you can remove <arg1> and <arg2> here and do something like this in the SearchTokens.py file:
`params = request.args.to_dict()` and then access arg1 and arg2 with params

Reply:
@kalenluciano yeah support for that isn't on this branch. On main, the workaround I used is you have to pass "all" for the arg you want to leave blank